plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/GlueIcebergTableOperations.java

```diff
@@ -132,6 +134,15 @@ protected void commitToExistingTable(TableMetadata base, TableMetadata metadata)
             // CommitFailedException is handled as a special case in the Iceberg library. This commit will automatically retry
             throw new CommitFailedException(e, "Failed to commit to Glue table due to concurrent updates: %s.%s", database, tableName);
         }
+        catch (ResourceNumberLimitExceededException e) {
```

Comment:
Let's write it as a multi-catch `catch` clause.

i think we can be as inclusive as possible, i.e. why not have EntityNotFoundException, InvalidInputException here?

Reply:
> i think we can be as inclusive as possible, i.e. why not have EntityNotFoundException, InvalidInputException here?

Added an extra catch for these use-cases. We don't want to wrap these exception in a `CommitFailedException` to perform retries.